prac_05/hex_colours.py

```diff
@@ -0,0 +1,14 @@
+COLOUR_TO_HEX = {"aliceblue": "#f0f8ff", "antiquewhite": "#faebd7", "black": "#000000", "blueviotlet": "#8a2be2",
+                 "brown": "#a52a2a", "burlywood": "#deb887", "cadetblue": "#5f9ea0"}
+print(COLOUR_TO_HEX)
+
+colour_name = input("Enter colour name: ")
+while colour_name != "":
+    if colour_name in COLOUR_TO_HEX:
+        print(colour_name, "is", COLOUR_TO_HEX[colour_name])
+    else:
+        print("Invalid colour name")
+    colour_name = input("Enter colour name: ")
+
+for key, value in COLOUR_TO_HEX.items():
```

Comment:
Is this meant to be "what's the key, what's the value". (something that Lindsay's been rather focused on, suggest more detail on variable names).